### ISSUE TITLE
Segmentation fault/access violation with MODFLOW-NWT

### ISSUE THREAD

**Issue by szipper** [modflowpy/flopy#320]
I am trying to convert an existing MODFLOW-2005 model to work with MODFLOW-NWT. However, I am running into an issue where I cannot get any model to run with MODFLOW-NWT!

I created the same basic script as this previous issue (#208): [SimpleNWT.py](https://github.com/szipper/FloPy_Tutorials/blob/master/SimpleNWT/SimpleNWT.py)

```
import flopy.modflow as mf
import platform

# what version of modflow to use?
modflow_v = 'mfnwt'  # 'mfnwt' or 'mf2005'

# where is your MODFLOW executable?
if (modflow_v=='mf2005'):
    if platform.system() == 'Windows':
        path2mf = 'C:/Users/Sam/Dropbox/Work/Models/MODFLOW/MF2005.1_12/bin/mf2005.exe'
    else: 
        path2mf = modflow_v
elif (modflow_v=='mfnwt'):
    if platform.system() == 'Windows':
        path2mf = 'C:/Users/Sam/Dropbox/Work/Models/MODFLOW/MODFLOW-NWT_1.1.4/bin/MODFLOW-NWT.exe'
    else:
        path2mf = modflow_v
        
# set up super simple model
ml = mf.Modflow(modelname="testmodel", exe_name=path2mf, version=modflow_v)
dis = mf.ModflowDis(ml)
bas = mf.ModflowBas(ml)
oc = mf.ModflowOc(ml)

# choose solver package depending on modflow version
if (modflow_v=='mf2005'):
    pcg = mf.ModflowPcg(ml)
elif (modflow_v=='mfnwt'):
    # linmeth has two matrix solver options (1 or 2)
    nwt = mf.ModflowNwt(ml)

ml.write_input()
ml.run_model()
```

If I run it on a Windows system using MODFLOW-NWT 1.1.4 downloaded from the [USGS website](https://water.usgs.gov/ogw/modflow-nwt/), I get an Access Violation error:
```
FloPy is using the following executable to run the model: C:/Users/Sam/Dropbox/Work/Models/MODFLOW/MODFLOW-NWT_1.1.4/bin/MODFLOW-NWT.exe

                                  MODFLOW-NWT-SWR1
    U.S. GEOLOGICAL SURVEY MODULAR FINITE-DIFFERENCE GROUNDWATER-FLOW MODEL
                             WITH NEWTON FORMULATION
                             Version 1.1.4 4/01/2018
                    BASED ON MODFLOW-2005 Version 1.12.0 02/03/2017

                    SWR1 Version 1.04.0 09/15/2016

 Using NAME file: testmodel.nam
 Run start date and time (yyyy/mm/dd hh:mm:ss): 2018/04/13 15:04:24

 Solving:  Stress period:     1    Time step:     1    Groundwater-Flow Eqn.
forrtl: severe (157): Program Exception - access violation
Image              PC        Routine            Line        Source
MODFLOW-NWT.exe    014254CA  Unknown               Unknown  Unknown
MODFLOW-NWT.exe    01438D3B  Unknown               Unknown  Unknown
MODFLOW-NWT.exe    01502DDF  Unknown               Unknown  Unknown
MODFLOW-NWT.exe    01531B72  Unknown               Unknown  Unknown
KERNEL32.DLL       75FA8654  Unknown               Unknown  Unknown
ntdll.dll          77B64B17  Unknown               Unknown  Unknown
ntdll.dll          77B64AE7  Unknown               Unknown  Unknown
```

If I run it on a Linux machine using `mfnwt 1.1.3` compiled using `pymake`, I get a segmentation fault error:
```
FloPy is using the following executable to run the model: /home/zipper/projects/def-tgleeson/zipper/MODFLOW/mfnwt

                                  MODFLOW-NWT-SWR1
    U.S. GEOLOGICAL SURVEY MODULAR FINITE-DIFFERENCE GROUNDWATER-FLOW MODEL
                             WITH NEWTON FORMULATION
                             Version 1.1.3, 8/01/2017
                    BASED ON MODFLOW-2005 Version 1.11.0 08/08/2013

                    SWR1 Version 1.04.0 09/15/2016

 Using NAME file: testmodel.nam
 Run start date and time (yyyy/mm/dd hh:mm:ss): 2018/04/13 14:54:47

 Solving:  Stress period:     1    Time step:     1    Groundwater-Flow Eqn.

Program received signal SIGSEGV: Segmentation fault - invalid memory reference.

Backtrace for this error:
#0  0x7F3F6FB35FE7
#1  0x7F3F6FB351E0
#2  0x7F3F6FE6E27F
#3  0x60A84F in rms_func_
#4  0x60DFA8 in gwf2nwt1fm_
#5  0x5A8253 in MAIN__ at MF_NWT.f:?
```

I'm no expert, but my impression is that the segmentation fault and the access violation are basically the same thing - a script is trying to access part of an array that doesn't exist, or something along those lines.

I am using flopy 3.2.9 on both machines. I have also tried:

- older versions of flopy (3.2.8, 3.2.3)
- messing with various flags in the NWT package (options, linmeth, etc)

I'm assuming there's something simple I'm missing like in issue #208, but am stumped at this point. Any suggestions?

The model works fine with MODFLOW-2005 on both the Linux and Windows machines:
```
(C:\Anaconda3) C:\Users\Sam\WorkGits\FloPy_Tutorials\SimpleNWT>python SimpleNWT.py
FloPy is using the following executable to run the model: C:/Users/Sam/Dropbox/Work/Models/MODFLOW/MF2005.1_12/bin/mf2005.exe

                                  MODFLOW-2005
    U.S. GEOLOGICAL SURVEY MODULAR FINITE-DIFFERENCE GROUND-WATER FLOW MODEL
                             Version 1.12.00 2/3/2017

 Using NAME file: testmodel.nam
 Run start date and time (yyyy/mm/dd hh:mm:ss): 2018/04/13 14:49:45

 Solving:  Stress period:     1    Time step:     1    Ground-Water Flow Eqn.
 Run end date and time (yyyy/mm/dd hh:mm:ss): 2018/04/13 14:49:45
 Elapsed run time:  0.004 Seconds

  Normal termination of simulation
```

**Comment by jdhughes-usgs** [comment]
You need a ups file for modflow-nwt.  You would also need a lpf or bcf package for modflow-2005.

**Comment by szipper** [comment]
Ah of course... I suspected it was something simple like this! Thanks.

For the sake of any future people that look at this issue, note that adding the UPW file to the [simple script](https://github.com/szipper/FloPy_Tutorials/blob/master/SimpleNWT/SimpleNWT.py) led to a different error (divide-by-zero), I'm guessing something to do with just using the defaults for every package. 

But [here is a different simple example](https://github.com/szipper/FloPy_Tutorials/blob/master/SquareWithWell-SteadyState/SquareWithWell-SteadyState.py) which worked fine with MODFLOW-NWT once I replaced LPF with UPW.
